api/app/responses/tests.py

```diff
@@ -45,43 +57,66 @@ def _seed_data(self):
         db.session.add_all(email_templates)
         db.session.commit()
 
-        # Add a user
+        # Add country
         test_country = Country('Indaba Land')
         _add_object_to_db(test_country)
 
+        # Add category
         test_category = UserCategory('Category1')
         _add_object_to_db(test_category)
 
+        # Add users to database
         other_user_data = self.user_data_dict.copy()
-        other_user_data['email'] = 'other@user.com'
+        other_user_data['email'] = self.other_user_email
         response = self.app.post('/api/v1/user', data=other_user_data)
         self.other_user_data = json.loads(response.data)
 
         response = self.app.post('/api/v1/user', data=self.user_data_dict)
         self.user_data = json.loads(response.data)
 
+        self.add_n_users(self.num_dummy_users)
+
         # Add application form data
-        self.test_event = self.add_event('Test Event', 'Event Description', date(2019, 2, 24), date(2019, 3, 24), 'NAGSOLVER')
+        self.test_event = self.add_event('Test Event', 'Event Description', date(2019, 2, 24), date(2019, 3, 24),
+                                         'NAGSOLVER')
         self.test_form = self.create_application_form(
-            self.test_event.id, True)
-        test_section = Section(
-            self.test_form.id, 'Test Section', 'Test Description', 1)
-        _add_object_to_db(test_section)
-        self.test_question = Question(self.test_form.id, test_section.id,
+                self.test_event.id, True)
+        self.test_section = Section(
+                self.test_form.id, 'Test Section', 'Test Description', 1)
+        _add_object_to_db(self.test_section)
+        self.test_question = Question(self.test_form.id, self.test_section.id,
                                       'Test Question Description', 'Test question placeholder', 1, 'Test Type', None)
         _add_object_to_db(self.test_question)
         self.test_question2 = Question(
-            self.test_form.id, test_section.id, 'Test Question 2', 'Enter something', 2, 'short-text', None)
+                self.test_form.id, self.test_section.id, 'Test Question 2', 'Enter something', 2, 'short-text', None)
         _add_object_to_db(self.test_question2)
 
+        # responses
         self.test_response = Response(
-            self.test_form.id, self.other_user_data['id'])
+                self.test_form.id, self.other_user_data['id'])
         _add_object_to_db(self.test_response)
 
         self.test_answer1 = Answer(
-            self.test_response.id, self.test_question.id, 'My Answer')
+                self.test_response.id, self.test_question.id, 'My Answer')
         _add_object_to_db(self.test_answer1)
```

Comment:
Can you update these to use your self.add_to_db function instead and remove the _add_object_to_db function